lib/cell-manager.coffee

```diff
@@ -0,0 +1,51 @@
+_ = require 'lodash'
+
+module.exports = CellManager =
+    removeAllBreakPoints: ->
```

Comment:
I feel code `Cell` might not be the best word.
@rgbkrk @willwhitney any suggestions from the native speaker?